principles/1_SecurityImplementation.md

```diff
@@ -108,7 +107,7 @@ policy compliance
 * Verify that all sensitive data is identified and classified into protection levels
 * Verify that all protection levels have an associated set of protection requirements, such as encryption requirements, integrity requirements, retention, privacy and other confidentiality requirements, and that these are applied in the architecture
 
-#### V1.9 Communications Architectural Requirements
+#### 1.9 Communications Architectural Requirements
 * Verify the application encrypts communications between components, particularly when these components are in different containers, systems, sites, or
```

Comment:
Verify the application encrypts communications between components, particularly when these components are in different containers, systems, sites, or cloud providers